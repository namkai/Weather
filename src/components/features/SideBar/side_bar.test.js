import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import { SideBar } from './index';

it('renders correctly', () => {
	const sidebar = shallow(<SideBar />);
	expect(toJson(sidebar)).toMatchSnapshot();
});
