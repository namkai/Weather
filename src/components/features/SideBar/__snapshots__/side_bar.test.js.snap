// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<SideBarLayout
  searchBar={
    <withState(withHandlers(pure(SearchBar)))
      fetchWeather={[Function]}
    />
  }
>
  <lifecycle(pure(Cities))
    clearStoredCities={[Function]}
    fetchWeather={[Function]}
    storedCities={[Function]}
    updateStoredCities={[Function]}
  />
</SideBarLayout>
`;
